packages/components/pages/f-account-info/src/components/AccountInfo.vue

```diff
@@ -137,23 +138,56 @@ export default {
 
     data () {
         return {
-            fields: {
-                emailAddress: null,
-                firstName: null,
-                lastName: null,
-                line1: null,
-                line2: null,
-                line3: null,
-                city: null,
-                postcode: null
-            },
-            tenantConfigs
+            fields: {},
```

Comment:
I suspect you will have lots of console errors because the fields mapped to the inputs aren't there